docs/knowledgebase/_index.md

```diff
@@ -0,0 +1,7 @@
+---
+title: "Knowledge Base"
+linkTitle: "Knowledge Base"
+weight: 900
+description: >
+  Articles and solutions on how to troubleshoot and resolve Ondat issues.
+---
```

Comment:
Should we add some text that if people have any questions they can contact support, or suggest additional articles through github issues?

Reply:
I think we should keep that information separate. We already mention instructions on how to contribute to documentation here > https://docs.ondat.io/docs/reference/contributing/ and also provide support information here > https://docs.ondat.io/docs/support/